curator-test-zk35/pom.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Some of our `pom.xml` have an Apache LICENSE header. I'm unclear where we should include it but a reminder.